src/main/java/bio/terra/cli/serialization/userfacing/UFWorkspaceUser.java

```diff
@@ -36,21 +36,21 @@ private UFWorkspaceUser(Builder builder) {
   /** Print out this object in text format. */
   public void print() {
     PrintStream OUT = UserIO.getOut();
-    List<String> rolesStr = roles.stream().map(IamRole::toString).collect(Collectors.toList());
+    List<String> rolesStr = roles.stream().map(Role::toString).collect(Collectors.toList());
     OUT.println(email + ": " + String.join(",", rolesStr));
```

Comment:
nit: you could do the join in your collect statement. collectors.joining or some such.

Reply:
done.